bazel/internal_python_rules.bzl

```diff
@@ -38,6 +40,7 @@ def internal_py_grpc_test(name, **kwargs):
         tests = [
             name + ".native",
             name + ".gevent",
+            name + ".typeguard",
```

Comment:
Similar to the logging test, this will increase test times and increase any existing flakiness. We need to keep a close eye on our tests after this PR is merged. If the flakiness or test duration turns out to be unacceptable, we may have to split things out to a separate Kokoro job somehow.

Reply:
We'll monitor this closely on release